extensions/ql-vscode/src/databases-ui.ts

```diff
@@ -348,6 +358,36 @@ export class DatabaseUI extends DisposableObject {
     }
   };
 
+  private handleSetDefaultTourDatabase = async (
+    progress: ProgressCallback,
+    token: CancellationToken,
+  ): Promise<void> => {
+    try {
+      if (workspace.workspaceFolders === undefined) {
+        throw new Error("No workspace folder is open.");
+      } else {
+        const codespaceRootFolderUri = workspace.workspaceFolders[0].uri;
```

Comment:
I think it's fine for us to say `workspace.workspaceFolders[0]`, since we control the associated tutorial workspace and we know that this is the right folder. 

Would it be helpful to add a comment mentioning that this specifically refers to https://github.com/github/codespaces-codeql?